docs/user-guide/layer-settings.md

```diff
@@ -1,120 +1,187 @@
+In particular, the user can choose between:
+
+* **Text**
+
+* **HTML**
+
+* **Properties**
+
+* **Template**
+
+!!!note
+    Without selecting any format here, the [Identify Tool](side-bar.md#identify-tool) will return the layers information with the format chosen in Map Settings (a [Burger Menu](menu-bar.md#burger-menu) option). Once a user specifies the information format in layers settings, instead, that format will take precedence over the map settings only for that specific layer.
+
+### Text
+
+An example of layer information in text format can be:
+
+<img src="../img/layer-settings/GFI_text.jpg" class="ms-docimage"  style="max-width:600px;"/>
+
+### HTML
+
+An example of layer information in HTML format can be:
+
+<img src="../img/layer-settings/GFI_html.jpg" class="ms-docimage"/>
+
+### Properties
+
+An example of layer information in properties format can be:
+
+<img src="../img/layer-settings/GFI_properties.jpg" class="ms-docimage"  style="max-width:600px;"/>
+
+### Templates
+
+In this case the user can customize the information format:
+
+<img src="../img/layer-settings/GFI_template.jpg" class="ms-docimage"  style="max-width:500px;"/>
+
+In particular, by clicking on the <img src="../img/button/edit_button.jpg" class="ms-docbutton"/> button, the following text editor appears:
+
+<img src="../img/layer-settings/edit_custom_format.jpg" class="ms-docimage"  style="max-width:600px;"/>
+
+In here it is possible to insert the text to be displayed through the [Identify Tool](side-bar.md#identify-tool), with the possibility to wrap the desired properties. <br>
+Let's make an example: we assume to have a layer where each record corresponds to a USA State geometry in the map. In the [Attribute Table](attributes-table.md) of this layer there's the `STATE_NAME` field that, for each record, contains a text value with the name of the State. <br>
+If the goal is to show, performing the [Identify Tool](side-bar.md#identify-tool), only the State name, an option could be to insert the following text on the Template text editor:
 
-* **Select** the HTML format, Then **Save** it.
+<img src="../img/layer-settings/GFI_template_ex.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-<img src="../img/layer-settings/html.jpg" class="ms-docimage"  style="max-width:500px;"/>
+In this case, by clicking on the map, the [Identify Tool](side-bar.md#identify-tool) returns:
 
-* **Click** on the layer. The info will be returned as a record (row) of a table corresponding to the info of that feature element of the layer.
+<img src="../img/layer-settings/GFI_template_ex1.jpg" class="ms-docimage"/>
 
-<img src="../img/layer-settings/html-1.jpg" class="ms-docimage"/>
+Using the `${properties.NAME_OF_THE_FIELD}` format, as you can see, it is possible to call dynamically the records of the chosen field matching the coordinates of the [Identify Tool](side-bar.md#identify-tool).
```

Comment:
Using the `${properties.NAME_OF_THE_FIELD}` syntax, MapStore is able to parse the response to the [Identify Tool](side-bar.md#identify-tool) request by matching the configured placeholder.